action.yml

```diff
@@ -85,6 +89,15 @@ runs:
         files: "${{ inputs.dockerfile }}"
         fail: true # fails the Action if Dockerfile is missing
 
+    - name: Prepare tags
+      shell: bash
+      id: tags
+      run: |
+        tags="${{ inputs.tags }}"
```

Comment:
Base on the definition https://github.com/sclorg/build-and-push-action/blob/main/action.yml#L24 it should be `${{ inputs.tag }}`

Reply:
Thank you for catching this, I will correct it.